server/src/main/java/org/candlepin/resource/DistributorVersionResource.java

```diff
@@ -170,14 +132,8 @@ public DistributorVersionDTO create(
         return this.translator.translate(curator.create(toCreate), DistributorVersionDTO.class);
     }
 
-    @ApiOperation(notes = "Updates a Distributor Version", value = "update")
-    @PUT
-    @Consumes(MediaType.APPLICATION_JSON)
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("/{id}")
-    public DistributorVersionDTO update(@PathParam("id") String id,
-        @ApiParam(name = "distributorVersion", required = true) DistributorVersionDTO dto,
-        @Context Principal principal) {
+    @Override
+    public DistributorVersionDTO update(String id, DistributorVersionDTO dto) {
```

Comment:
One more thing I forgot (sorry, I know I'm asking for changes multiple times instead of all at once, but it's hard to keep in mind everything that needs to be in line. I'll try to take some notes of a complete list of things to look for in each openapi port PR review from now on!): 

Now that we've marked the DistributorVersionCapabilityDTO's name field as `required: true` and `minLength: 1`, this is going to generate some validation annotations, but we still need to manually run the validation (with `DTOValidator`) on the `created/update` endpoints with: `validator.validateConstraints(dto);`. Also, the old DTO is validating that none of the capability DTOs set by the setCapabilities setter is null, so we also have to check this ourselves using `validator.validateCollectionElementsNotNull(dto::getCapabilities);`

Reply:
Done.